Week-10/url-shortner/src/Components/Header.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+
+function Header() {
+  return (
+    <div className='header'>
+      <div>
+        <h2>Shortly</h2>
+      </div>
+      <div className='main-functions'>
+        <Link to="/ab" >Features</Link>
+        <Link to="/ab" >Pricing</Link>
+        <Link to="/ab" >Resources</Link>
+      </div>
```

Comment:
Instead of hardcoding links can we put them into one file and use it from there.